crates/mdman/doc/out/mdman.md

```diff
@@ -54,8 +54,8 @@ man page:
 
 <dl>
 
-<dt><code>-t</code> <em>type</em></dt>
-<dd>Specifies the output type. The following output types are supported:</p>
+<dt class="option-term" id="option-mdman--t"><a class="option-anchor" href="#option-mdman--t"></a><code>-t</code> <em>type</em></dt>
```

Comment:
These seem to be some unrelated changes?

Reply:
Yes, I thought we need to update the man page if we are updating any details. That's why this file got updated.
No worries I'll remove these changes. 

Thanks for your response.